script.js

```diff
@@ -195,15 +502,49 @@ import getDishByMenu from "./models/dishModel.js";
       renderMenuItems(state.activeMenuItems[i]);
     }
 
+    renderCartCount();
     for (let i = 0; i < state.cart.length; i++) {
       renderCartItems(state.cart[i]);
     }
+    renderCartSummary();
+    renderCheckoutButton();
+    renderEmptyCart();
   }
 
   // function onUnmount() {
```

Comment:
get rid of the comments, make the PR clean.

Reply:
All comments are removed.